eng/Generate.ps1

```diff
@@ -292,7 +292,9 @@ if ($reset -or $env:TF_BUILD)
 if (!$noBuild)
 {
     Invoke "dotnet build $autoRestPluginProject"
-    Invoke-CadlSetup
+    if (!$fast) {
+        Invoke-CadlSetup
```

Comment:
Fast path doesn't invoke generation of `cadl.json`, so we don't need to setup CADL

Reply:
In fact, I think we should make this an explicit command. Even at full rebuild we don't need it more than once.

Reply:
!Fast path, we need to invoke generation of `cadl.json` and it need to re-build setup CADL, so I think we need to remain previous version
if (!$noBuild)
{
    Invoke "dotnet build $autoRestPluginProject"
    if (!$fast -and  $cadlCou]nt -gt 0) {
        Invoke-CadlSetup
   }
}